terraso_backend/apps/shared_data/views.py

```diff
@@ -66,6 +73,15 @@ def post(self, request, **kwargs):
         return JsonResponse(data_entry.to_dict(), status=201)
 
 
+def is_valid_shared_data_type(files):
+    for file in files:
+        extension = Path(file.content_type)
+        type = "." + extension.parts[-1]
+        if type not in DATA_ENTRY_ACCEPTED_EXTENSIONS:
+            return False
+    return True
```

Comment:
Suggestion: It seems that you should be able to do something like this here:
```suggestion
def is_valid_shared_data_type(files):
    return all(Path(file.content_type).suffix in DATA_ENTRY_ACCEPTED_EXTENSIONS for file in files)
```

Reply:
@josebui I think `.suffix` only gets the file extension so any string after `.`, I will use `.name` instead